docs/usage.md

```diff
@@ -123,4 +123,4 @@ function eg_log_action_data() {
 add_action( 'eg_midnight_log', 'eg_log_action_data' );
 ```
 
-For more details on all available API functions, and the data they accept, refer to the [API Reference](/api/).
+Note that the `as_has_scheduled_action()` function was added in 3.3.0: if you are using an earlier version, you should substitute `as_next_scheduled_action()` instead. For more details on all available API functions, and the data they accept, refer to the [API Reference](/api/).
```

Comment:
This might be a personal preference but I find the use of `substitute` here confusing. I am not sure if that means I indeed should substitute `as_next_scheduled_action()` with `as_has_scheduled_action()` or the other way around.

So I would prefer we change it to `use` to be more clear? Again it could be just me that is confused about it.

Reply:
Good call—I agree that reads better. Updated :+1: